examples/agent-server/client.py

```
@@ -1,23 +1,12 @@
-# URL = "http://localhost:8000/chat"
-CELESTO_API_KEY = os.environ.get("CELESTO_API_KEY")
-
-headers = {
-    "Authorization": f"Bearer {CELESTO_API_KEY}",
-    "Content-Type": "application/json",
-}
+# Point this at wherever you are running main.py. The app `agent.serve()` builds
+# is a plain FastAPI app with no authentication, so no credentials are needed.
+URL = "http://localhost:8000/chat"
 
 response = requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)